src/util.ts

```diff
@@ -347,8 +347,8 @@ export class Util {
         }
 
         let haveLevel = 0;
-        if (res && res.users_default) {
-            haveLevel = res.users_default;
+        if (res && res.users_default || 0) {
```

Comment:
This change does nothing.

Reply:
My bad, I will remove the || 0 in this line again